dbms/src/Flash/Mpp/ExchangeReceiver.cpp

```diff
@@ -411,7 +411,8 @@ ExchangeReceiverBase<RPCContext>::ExchangeReceiverBase(
     size_t max_streams_,
     const String & req_id,
     const String & executor_id,
-    uint64_t fine_grained_shuffle_stream_count_)
+    uint64_t fine_grained_shuffle_stream_count_,
+    bool is_tiflash_storage_receiver_)
```

Comment:
This naming is vague: Is it "an receiver on tiflash compute node receiving data from tiflash storage node" or "an receiver on tiflash storage node"?

Reply:
Changed to `is_receiver_for_tiflash_storage`